libcaf_core/caf/actor_system_config.hpp

```diff
@@ -252,6 +252,13 @@ class actor_system_config {
   size_t work_stealing_relaxed_steal_interval;
   size_t work_stealing_relaxed_sleep_duration_us;
 
+  // -- config parameters for the logger ---------------------------------------
+
+  std::string logger_filename;
```

Comment:
Why support only `[NODE]` as dynamic placeholder? Seems inconsequential to me. We could instead give users more fine-grained control over the filename and set the default to `actor_log_[PID]_[TIMESTAMP]_[NODE].log`.